src/main/java/tools/jackson/core/json/JsonFactory.java

```diff
@@ -183,14 +183,32 @@ public JsonFactoryBuilder rebuild() {
 
     /**
      * Main factory method to use for constructing {@link JsonFactory} instances with
-     * different configuration.
-     *
+     * different configuration. The builder returned uses default settings more closely
```

Comment:
I think this Javadoc is accidentally swapped with next one?

Reply:
fixed just now